src/tls_wrap.cc

```diff
@@ -701,8 +701,15 @@ int TLSWrap::DoWrite(WriteWrap* w,
 
   size_t length = 0;
   size_t i;
-  for (i = 0; i < count; i++)
+  size_t nonzero_i = 0;
+  size_t nonzero_count = 0;
```

Comment:
Can we maybe use `nonemtpy_...` or similar instead of `nonzero_...`?

Reply:
changed in 3de879f